misp/plugin.spec.yaml

```diff
@@ -1198,7 +1198,7 @@ actions:
         type: string
         required: false
         enum:
-        - Don't search on
+        - Do not search on
```

Comment:
Need a major version bump for these too since the values change. Any workflow depending on the previous value will not work if the plugin is upgraded.